### DISCUSSION TITLE
寫錯字有差嗎

### DISCUSSION THREAD

**Discussion by roma0323** [1101-GS4719-C/Python-Community-GS4719-C#1546] [General]
我身邊大部分朋友都不太在呼錯字，但有位特別在乎，如果跟他聊天寫了錯字他會特別抓出來訂正，之前跟他掙論過文字就是思想傳遞的媒介，如果打了錯字對方還是能夠理解，那這個字便有好好完成他的任務，所以我應該也是覺得即便錯字但只要對方懂就不算錯字啦

**Comment by aa103967622** [comment]
我之前做過一個社會學實驗，去測試說給受試者一篇用字和用法都有錯的文章，並讓受試者唸出來，觀察途中是否有出現發現錯字卻沒有指正繼續唸下去的情況，並且他本人並沒有察覺，這代表錯字不太影響人們的閱讀！

**Comment by Chiuhsianglin** [comment]
我認為錯字每個人對不同的字都有不同的看法，像是「在」跟「再」，我看到有人打錯這個就會蠻在意的，相反的如果是兩個字中的一個字打錯，我可能就覺得還好。

**Comment by lyien1021** [comment]
我覺得我是不太在意別人打錯字，反而會注意自己有沒有打錯字，不知道為什麼就不喜歡自己打錯字的感覺

**Comment by zhiron544** [comment]
我覺得寫錯字的嚴重程度取決於情境和對象，有些場合和人需要更正式和正確的文字，有些則可以比較隨意和寬容。我認為文字是一種表達工具，但也是一種文化和習慣，所以我會盡量避免寫錯字，也會尊重別人的文字風格。

**Comment by 110302001** [comment]
我覺得像聊天打錯字這種我就蠻不在意，也不會特別糾正，因為現在大家打字很快時的確很容易按錯，或手機自動選字常常也會錯誤，我就覺得沒什麼好在意的，也真的不嚴重，除非正式文件、卡片、考卷等就真的不能錯字
